### PR TITLE
Add AUR installation to readme.md

### PR DESCRIPTION
**Pull request by veronikazaglotova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ git clone --depth=1 https://github.com/instantOS/instantWM.git
 cd instantWM
 ./build.sh
 ```
+
+Or install from AUR:
+
+```
```
Comment (veronikazaglotova) [inline README.md:23]: Wow cool button didn't know it was there

**Comment by con-f-use** [comment]
Not an Archlinux person, but if the package is on AUR, shoudn't one install it using a package manager that supports AUR. Otherwise, there is no conceptual difference between cloning from this repo and cloning from the git repo hostet by the AUR.

**Comment by veronikazaglotova** [comment]
> shoudn't one install it using a package manager that supports AUR

What you're talking about is called "AUR helper" which wraps around pacman (the Arch package manager). E.g. to install `instantwm` I only have to type `yay -S instantwm` instead of all that stuff. Sure, this is better, but not everyone uses the same AUR helper. After that, I can update it with `yay -Syu` or just simply `yay` with all the other packages on my system.

> this PR would just add unnecessary redundancy to the README

First of all, `build.sh` runs `sudo make install` which is considered harmful because it's hard to uninstall the program that was installed this way. A lot of the times even after doing `make uninstall` there are leftover files somewhere in your system.  Installing from AUR solves this issue, because you `makepkg -si` installs InstantWM with pacman, e.g. pacman is fully aware of all the files that were installed and you can later uninstall instantwm with `pacman -Rcns`. Secondly, installing a package from AUR allows you to update instantWM with your AUR helper, as I explained above. 
It's not obvious that there's an AUR package, every newbie in the Linux world using Manjaro/Antergos/Archlabs or any other user-friendly Arch-based distribution will lose all mentioned advantages of using AUR if we don't tell them to do so.

**Comment by veronikazaglotova** [comment]
A bit of anecdotal evidence. I had such an issue myself, but it was just a little different. I was using Ubuntu 16.04 and wanted to start programming with Python. I opened python.org and downloaded the source code for Python 3, then installed it with make install. It replaced Python2, and after that I couldn't open gnome terminal. That vas because I didn't know that I can install Python with apt-get.

**Comment by con-f-use** [comment]
Sure, and I'm saying, just put the `yay` command in the readme. People, who don't use yay, will know the equivalent method on their package manager or know to use makepkg, and the others (who are in the majority, btw.) will be the happier. It just feels superfluous.

**Comment by veronikazaglotova** [comment]
Oh ok. I'll make a new pull request I guess

File: README.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ git clone --depth=1 https://github.com/instantOS/instantWM.git
 cd instantWM
 ./build.sh
```
Comment (con-f-use) [inline README.md:18]: how about  this

```suggestion
./build.sh  # makepkg -si  # if you cloned from aur.archlinux.org/instantwm.git
```

and then delete the whole section below.

**Comment by paperbenni** [comment]
Sorry, I thought I commented on this but I guess the GitHub app just wasn't having it. 
I won't put anything AUR related in any official places yet. The AUR versions are outdated and some dependencies actually got removed from the AUR. I will have to clear that up some time but at the moment instantOS does a good job at keeping me busy already. instantOS is bit like pantheon. Sure, you can run it on pretty much any Unix system, but outside of the official OS (or community ports like @con-f-use 's nix port) you're more on your own. I won't imply that this isn't the case when it at this time still is.

**Pull request merged**